src/test/java/uk/gov/hmcts/reform/civil/handler/callback/user/FullRemissionHWFCallbackHandlerTest.java

```diff
@@ -0,0 +1,74 @@
+package uk.gov.hmcts.reform.civil.handler.callback.user;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.beans.factory.annotation.Autowired;
+import uk.gov.hmcts.reform.ccd.client.model.AboutToStartOrSubmitCallbackResponse;
+import uk.gov.hmcts.reform.civil.callback.CallbackParams;
+import uk.gov.hmcts.reform.civil.callback.CallbackType;
+import uk.gov.hmcts.reform.civil.enums.FeeType;
+import uk.gov.hmcts.reform.civil.handler.callback.BaseCallbackHandlerTest;
+import uk.gov.hmcts.reform.civil.model.CaseData;
+import uk.gov.hmcts.reform.civil.model.Fee;
+import uk.gov.hmcts.reform.civil.model.citizenui.HelpWithFeesDetails;
+
+import java.math.BigDecimal;
+
+import static uk.gov.hmcts.reform.civil.callback.CaseEvent.FULL_REMISSION_HWF;
+import static org.assertj.core.api.Assertions.assertThat;
+
+@ExtendWith(MockitoExtension.class)
+public class FullRemissionHWFCallbackHandlerTest extends BaseCallbackHandlerTest {
+
+    private FullRemissionHWFCallbackHandler handler;
+    @Autowired
```

Comment:
Autowired not required, Initialize Object mapper in beforeEach and can use same instance for FullRemissionHWFCallbackHandler and for converting json